packages/atomic/src/components/search/atomic-generated-answer/atomic-generated-answer.tsx

```diff
@@ -80,14 +85,22 @@ export class AtomicGeneratedAnswer implements InitializableComponent {
    */
   @Prop() answerStyle: GeneratedAnswerStyle = 'default';
 
+  /**
+   * The answer is collapsed when the text is longer than 250px
+   */
+  @Prop() collapsible?: boolean;
+
   @AriaLiveRegion('generated-answer')
   protected ariaMessage!: string;
 
   private generatedAnswerCommon!: GeneratedAnswerCommon;
+  private fullAnswerHeight?: number;
+  private maxCollapsedHeight = 250;
```

Comment:
This is not so much relative to the height of the viewport, but more related to the height of the content of the answer.
This could be transformed in a max height in `em` for example, but it would make it very hard I think to calculate in js instead of in CSS...